docs/gettsim_objects/input_variables.md

```diff
@@ -496,3 +496,21 @@ Months of subjection to compulsory insurance in the 5 years before claiming unem
 insurance
 
 Type: float
+
+## `kind_unterh_brutto_m`
+
+Monthly gross child alimony payments
```

Comment:
If we keep this variable, we need to make clearer what is meant by brutto netto here.